pipeline-runner/src/data-ingest/5_Upload-to-aws.r

```diff
@@ -15,7 +15,7 @@ create_samples_table <- function(config, experiment_id, project_id) {
                              row.names = 'samples')
 
   samples <- row.names(df_prefiltered)
-  samples_table$ids = paste0("sample-", samples)
+  samples_table$ids <- lapply(samples, function(x) paste0("sample-", x, sep = ""))
```

Comment:
I'm pretty sure I didn't make this change. It's carried over from Martin's branch